data/mods/Boats/whaleys_boats.json

```diff
@@ -0,0 +1,147 @@
+[
+  {
+    "//": "Place the following vehicles as boat.json in /data/json/vehicles/",
+    "type": "vehicle",
+    "id": "canoe",
+    "name": "canoe",
+    "blueprint": [ [ "<OO>" ] ],
+    "parts": [
+      { "x": 0, "y": 0, "part": "frame_wood_horizontal" },
+      { "x": 0, "y": 0, "part": "seat" },
+      { "x": 0, "y": 0, "part": "hand_paddles" },
+      { "x": 0, "y": 0, "part": "boat_board" },
+      { "x": 1, "y": 0, "part": "frame_wood_horizontal" },
+      { "x": 1, "y": 0, "part": "boat_board" },
+      { "x": 1, "y": 0, "part": "seat" },
+      { "x": 2, "y": 0, "part": "frame_wood_horizontal" },
+      { "x": 2, "y": 0, "part": "boat_board" },
+      { "x": -1, "y": 0, "part": "frame_wood_horizontal" },
+      { "x": -1, "y": 0, "part": "boat_board" }
+    ]
+  },
+  {
+    "type": "vehicle",
+    "id": "raft",
+    "name": "raft",
+    "blueprint": [ [ "OO" ], [ "OO" ] ],
+    "parts": [
+      { "x": 0, "y": 0, "part": "frame_wood_cross" },
+      { "x": 0, "y": 0, "part": "seat" },
+      { "x": 0, "y": 0, "part": "hand_paddles" },
+      { "x": 0, "y": 0, "part": "boat_board" },
+      { "x": 1, "y": 0, "part": "frame_wood_cross" },
+      { "x": 1, "y": 0, "part": "boat_board" },
+      { "x": 1, "y": 1, "part": "frame_wood_cross" },
+      { "x": 1, "y": 1, "part": "boat_board" },
+      { "x": 0, "y": 1, "part": "frame_wood_cross" },
+      { "x": 0, "y": 1, "part": "boat_board" }
+    ]
+  },
+  {
+    "//": "Place the following in vehicle_groups.json in /data/json/",
+    "type": "vehicle_group",
+    "id": "boatrent",
+    "vehicles": [ [ "canoe", 2000 ], [ "raft", 2000 ] ]
+  },
+  {
+    "//": "Place the following in recipe_vehicle.json in /data/json/recipes/",
```

Comment:
Isn't something supposed to be done with this?

Reply:
Yes, but later when boat vehicles would go to mainline from mod.